sdk/appconfiguration/arm-appconfiguration/package.json

```diff
@@ -37,12 +37,8 @@
     "@types/node": "^18.0.0",
     "@vitest/browser": "^2.1.8",
     "@vitest/coverage-istanbul": "^2.1.8",
-    "cross-env": "^7.0.2",
     "dotenv": "^16.0.0",
-    "mkdirp": "^3.0.1",
     "playwright": "^1.49.1",
-    "rimraf": "^5.0.0",
```

Comment:
rimraf is used in the `clean` command

Reply:
It's vendored through dev-tool, so it doesn't need rimraf locally, and instead dev-tool uses its own copy of rimraf.  Same with cross-env or any other tool.